src/test/java/strategy/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,18 @@
+package strategy;
+
+import db.FruitsDao;
+import db.GenericDao;
+import models.Fruit;
+import org.junit.Test;
+
+public class PurchaseOperationHandlerTest {
+    private static final GenericDao<Fruit, Integer> fruitsDao = new FruitsDao();
+    private static final AdditionalOperationHandler additionalOperationHandler
+            = new AdditionalOperationHandler(fruitsDao);
+    private static final Fruit izir = new Fruit("inzir");
+
+    @Test(expected = RuntimeException.class)
+    public void changeBalance_NotOk() {
+        additionalOperationHandler.changeBalance(izir.getName(), -100);
+    }
+}
```

Comment:
let's add a test when everything works well, we change balance, then go to storage and check the result
then we clear the storage